code/game/machinery/doors/firedoor.dm

```diff
@@ -920,6 +920,16 @@
 				return
 	return ..()
 
+/obj/machinery/door/firedoor/attackby_secondary(obj/item/weapon, mob/user, params)
+	. = ..()
+	if(!isliving(user))
+		return
+	var/mob/living/living_user = user
+	if(!(living_user.combat_mode || istype(weapon, /obj/item/crowbar)))
+		return
+	try_to_crowbar_secondary(weapon, living_user)
+	return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
```

Comment:
This is already supposed to be [a thing](https://github.com/tgstation/tgstation/pull/56960), but it's broken because tool acts are no longer called when a mob is in combat mode

You should put this in `/obj/machinery/door/item_interaction_secondary`:

```dm
/obj/machinery/door/item_interaction_secondary(mob/living/user, obj/item/tool, list/modifiers)
    // allows you to crowbar doors while in combat mode
    if(user.combat_mode && tool.tool_behavior == TOOL_CROWBAR)
        return crowbar_act_secondary(user, tool)
    return ..()